contrib/completion/bash/docker

```diff
@@ -334,14 +334,26 @@ _docker_diff() {
 
 _docker_events() {
 	case "$prev" in
-		--since)
+		--filter|-f)
+			COMPREPLY=( $( compgen -S = -W "container event image" -- "$cur" ) )
+			compopt -o nospace
+			return
+			;;
+		--since|--until)
+			return
+			;;
+	esac
+
+	case "${words[$cword-2]}$prev=" in
```

Comment:
I'm not sure I understand what this even means, so a comment would definitely be prudent IMO.